### PR TITLE
basement escape oh god

### PR DESCRIPTION
**Pull request by NatalieCrazy**

### CODE REVIEW

**Comment by MonkeysWithPie** [comment]
lucky for everyone the chance is 0.01% so nobody's leaving any time soon

**Comment by MonkeysWithPie** [comment]
wait what happens if someone escapes the basement twice, do they just go to heaven

**Review by milenakos — CHANGES_REQUESTED** [review]
newlines

File: main.py [hunk 1]
```diff
@@ -3235,8 +3235,8 @@ async def claim_reward(user, channeley, type):
             await channeley.send(f"<@{user.user_id}>", embed=embedVar, view=view)
     except Exception:
         pass
-
-
+    if randint(0, 10000) = 236:
```
Comment (frinkifail) [inline main.py:3238]: `randint(...) == 236`

**Review by frinkifail — CHANGES_REQUESTED** [review]

**Comment by NatalieCrazy** [comment]
oki done

**Review by milenakos — COMMENTED** [review]
wont crash

**Comment by milenakos** [comment]
fuck erong hutton

**Review by milenakos — APPROVED** [review]

**Review by frinkifail — APPROVED** [review]

File: main.py [hunk 2]
```diff
@@ -3236,6 +3236,8 @@ async def claim_reward(user, channeley, type):
     except Exception:
         pass
 
+    if randint(0, 10000) == 236:
```
Comment (milenakos) [inline main.py:3239]:
```suggestion
    if random.randint(0, 10000) == 236:
```

**Review by milenakos — CHANGES_REQUESTED** [review]
nvm it will

File: main.py (hunk 2 above)
Comment (frinkifail) [reply main.py:3239]: i thought randint was already imported like `from random import randint` :staring_cat:

Comment (milenakos) [reply main.py:3239]: it was until some commits ago

Comment (NatalieCrazy) [reply main.py:3239]: im going to airstrike your house from orbit

**Review by milenakos — APPROVED** [review]
wont crash 4 sure now

File: main.py (hunk 2 above)
Comment (frinkifail) [reply main.py]: agreed

**Comment by NatalieCrazy** [comment]
we getting a merge or nah

**Comment by milenakos** [comment]
idk

**Comment by NatalieCrazy** [comment]
so real for that one

**Comment by MonkeysWithPie** [comment]
damn i wanted to see people hit a 1 in 100000000 and go to heaven

**Pull request closed without merging**